docs/_ug/commandSummary/StatisticsCommands.md

```diff
@@ -0,0 +1,27 @@
+<!-- markdownlint-disable-file first-line-h1 -->
+
+<!-- ===== DECLARE VARIABLES ===== -->
+<!-- markdownlint-disable -->
+{% capture stats %}{% include_relative _ug/commandSummary/statisticsCommands/stats.md %}{% endcapture %}
+
+{% assign stats = stats | markdownify %}
+
+{% capture statsexample %}{% include_relative _ug/commandSummary/statisticsCommandsExamples/stats.md %}{% endcapture %}
+
+{% assign statsexample = statsexample | markdownify %}
```

Comment:
Since it's only this one table row, let's clean up the code a bit:

```suggestion
{% capture stats %}{% include_relative _ug/commandSummary/statisticsCommands/stats.md %}{% endcapture %}
{% assign stats = stats | markdownify %}
{% capture statsexample %}{% include_relative _ug/commandSummary/statisticsCommandsExamples/stats.md %}{% endcapture %}
{% assign statsexample = statsexample | markdownify %}
```